src/workers/postUpdated.ts

```diff
@@ -255,10 +255,25 @@ const updatePost = async ({
   content_type = PostType.Article,
 }: UpdatePostProps) => {
   const postType = contentTypeFromPostType[content_type];
-  const databasePost = await entityManager
+  let databasePost = await entityManager
     .getRepository(postType)
     .findOneBy({ id });
 
+  // If we don't find the post, we need to check if it's a youtube video and
+  // try to find it again as an article
```

Comment:
This comment is outdated now since we do it in generic way?

Reply:
Yes it is 😬